.github/workflows/validate-projects.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v1
       - name: Set up Python 3.7
-        uses: s-weigand/setup-conda@v1.0.3
+        uses: s-weigand/setup-conda@v1
```

Comment:
a lot of GitHub Actions thing keep this major-version-only tag up to date. `@v1` is always updated to the latest release in the `1.x` series.

https://github.com/s-weigand/setup-conda/releases

I think this is better than bumping from 1.0.3 to 1.0.5 for these GitHub Actions, so we keep picking up bugfixes.

Reply:
Yep looks good.